src/pages/dho/Plan.vue

```diff
@@ -165,7 +170,14 @@ export default {
 
   },
 
-  created () {
+  async beforeCreate () {
+    // Hardcoded https://telos.greymass.com so we can see the mainnet hypha to usd convertion
+    const hyphaTokensSaleUtil = new HyphaTokensSaleUtil('https://telos.greymass.com', 'http://api-tokensale.hypha.earth')
```

Comment:
@pugakn can you move this to the function? something like `getUSDPerHypha`? Also I am not sure about hardcoding greymass because you know that something we change production? If in the future we need to change its not going to be just inside `.env`